Source/WebInspectorUI/UserInterface/Models/SourceCodePosition.js

```diff
@@ -25,10 +25,10 @@
 
 WI.SourceCodePosition = class SourceCodePosition
 {
-    constructor(lineNumber, columNumber)
+    constructor(lineNumber, columnNumber)
```

Comment:
Wow! This existed since the first commit nearly 10 years ago. Do we have other `columNumber` typos?

Reply:
ikr.  not that i could find :)